eahub/config/urls.py

```diff
@@ -52,3 +52,4 @@
 ]
 
 handler404 = "eahub.base.views.page_not_found"
+handler410 = "eahub.base.views.gone"
```

Comment:
This doesn't do anything; these are exception handlers and there are no exceptions in Django that result in 410 responses.

Reply:
I wanted to split this in 2 moves, but you're right. I'll just apply this to https://github.com/rtcharity/eahub.org/issues/603

Reply:
No, I mean there's no way this can ever do anything, because there's no such thing in Django as a 410 exception and you can't add one. Which exceptions return which status codes is hardcoded into the framework.